.github/actions/cdk-deploy/action.yml

```diff
@@ -9,6 +9,18 @@ inputs:
     required: false
     type: string
     default: "."
+  script_path:
+    required: true
+    type: string
+  project_prefix:
+    required: true
+    type: string
+  stage:
+    required: true
+    type: string
+
+
+
```

Comment:
why so many newlines lol

Reply:
Because they are free loool. I will fix that